spec/system/assessor_interface/authentication_spec.rb

```diff
@@ -4,6 +4,7 @@
 
 RSpec.describe "Assessor authentication", type: :system do
   it "allows signing in and signing out" do
+    FeatureFlags::FeatureFlag.deactivate(:enable_login_with_azure)
```

Comment:
Could we move these in to a `given_sign_in_with_active_directory_is_deactive` method, just to match the convention of these system tests.